README.md

```diff
@@ -26,8 +27,7 @@ into the `vars.tf` file where you can change them as you need.
 - An OpenStack Deployment where you want to launch VGCN
 - API access to this OpenStack
 - [Terraform](https://www.terraform.io/intro/getting-started/install.html)
-- The latest VGGP image ([from us](https://usegalaxy.eu/static/vgcn/), or [compiled yourself](https://github.com/usegalaxy-eu/vgcn/tree/passordless))
-- (Optional) A Galaxy instance which will use this cluster
+- The latest VGGP image ([here](https://usegalaxy.eu/static/vgcn/))
```

Comment:
VGCN or what is VGGP? :)

Reply:
Actually I don't know why VGCN images are labelled as "vggp"